tests/acceptance/bootstrap/GraphContext.php

```diff
@@ -2978,6 +2978,38 @@ public function theActivitiesShouldBeInTheFollowingOrder(TableNode $table): void
 		}
 	}
 
+	/**
+	 * @When /^user "([^"]*)" lists the activities of (?:folder|file) "([^"]*)" from space "([^"]*)" with (depth|limit) "([^"]*)" and sort "(asc|desc)" using the Graph API/
+	 *
+	 * @param string $user
+	 * @param string $resource
+	 * @param string $spaceName
+	 * @param string $filterType
+	 * @param string $filterValue
+	 * @param string $sortType
+	 *
+	 * @return void
```

Comment:
I think we can add `@throws` too
```suggestion
	 * @return void
	 * @throws GuzzleException
```